### PR TITLE
add website contribution link

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan < bill@isovalent.com >

For when https://github.com/cilium/cilium.io/pull/11 is merged

### CODE REVIEW

**Comment by joestringer** [comment]
@xmulligan if you configure your git user / email correctly, you should be able to just `git commit --amend -s` to add your signoff & repush to address the complaint from the MLH.

**Review by aditighag — APPROVED** [review]
Thanks! Please fix up sign-off.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 
+## To Cilium/Hubble
+
 See the [Developer / Contributor
 Guide](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/) for detailed information on
 how to contribute, get started and find good first issues.
+
+## To the website
+
+Please see the [website contributing guide](https://github.com/cilium/cilium.io/CONTRIBUTING.md) for detailed information on how to add blogs, trainings, and other resources
```
Comment (sayboras) [inline CONTRIBUTING.md]: for after https://github.com/cilium/cilium.io/pull/11 is merged

Should the link be https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md istead ?

**Comment by xmulligan** [comment]
closing in favor of #18940

**Pull request closed without merging**